packages/cyberstorm/src/components/BreadCrumbs/BreadCrumbs.tsx

```diff
@@ -3,41 +3,133 @@ import React, { PropsWithChildren } from "react";
 import { faHouse } from "@fortawesome/pro-regular-svg-icons";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import styles from "./BreadCrumbs.module.css";
-import { IndexLink } from "../Links/Links";
+import {
+  CommunitiesLink,
+  CommunityLink,
+  IndexLink,
+  PackageLink,
+  TeamsLink,
+} from "../Links/Links";
+
+type PageTitleCrumbProps = PropsWithChildren<{
+  pageTitle: string;
+}>;
 
 type BreadCrumbsProps = PropsWithChildren<{
-  excludeHome?: boolean;
+  pageTitle: string;
 }>;
 
-export function BreadCrumbs(props: BreadCrumbsProps) {
-  const children = React.Children.toArray(props.children);
-  if (!props.excludeHome) {
-    children.unshift(<DefaultHomeCrumb />);
-  }
-
-  const nodes = children.map((node, index) => {
-    const isLast = index == children.length - 1;
-    return (
-      <div
-        key={index}
-        className={`${index == 0 ? styles.outer__start : ""} ${styles.outer} ${
-          isLast ? styles.outer__end : ""
-        }`}
-      >
-        <div className={`${styles.inner} ${isLast ? styles.inner__end : ""}`}>
-          {node}
-        </div>
-      </div>
-    );
-  });
+interface PackageBreadCrumbsProps extends BreadCrumbsProps {
+  community: string;
+}
 
-  return <div className={styles.root}>{nodes}</div>;
+interface PackageDependantsBreadCrumbsProps {
+  community: string;
+  packageName: string;
+  packageNameSpace: string;
 }
 
 export function DefaultHomeCrumb() {
   return (
-    <IndexLink>
-      <FontAwesomeIcon fixedWidth icon={faHouse} className={styles.home} />
+    <IndexLink className={`${styles.outer} ${styles.outer__start}`}>
+      <div className={styles.inner}>
+        <FontAwesomeIcon fixedWidth icon={faHouse} className={styles.home} />
+      </div>
     </IndexLink>
   );
 }
+
+export function PageTitleCrumb(props: PageTitleCrumbProps) {
+  const { pageTitle } = props;
+  return (
+    <div className={`${styles.outer} ${styles.outer__end}`}>
+      <div className={`${styles.inner} ${styles.inner__end}`}>{pageTitle}</div>
+    </div>
+  );
+}
+
+export function HomeOnlyBreadCrumbs() {
```

Comment:
`HomeOnlyBreadCrumbs` and `TitleOnlyBreadCrumbs` *might* be worth keeping even if we don't end up creating separate BreadCrumbs components for most views.